src/views/Network.js

```diff
@@ -8,6 +7,7 @@ import { ROUTE_NAMES } from '../lib/routeNames';
 import { useHistory } from '../store/history';
 import { useNetwork } from '../store/network';
 import View from 'components/View';
+import { ForwardButton } from 'components/Buttons';
 
 const kNetworkOptions = [
```

Comment:
Don't worry about touching up these "deprecated" screens too much.

Reply:
@Fang- yeah, will nuke those for #174 once that's merged today